numpy/linalg/lapack_lite/dlapack_lite.f.patch

```diff
@@ -0,0 +1,239 @@
+diff --git a/dlapack_lite.f b/dlapack_lite.f.patched
+--- a/dlapack_lite.f
++++ b/dlapack_lite.f.patched
+@@ -13161,10 +13161,11 @@
+                VII = V( I, I )
+                V( I, I ) = ONE
+                IF( LSAME( STOREV, 'C' ) ) THEN
+ !                 Skip any trailing zeros.
++                  LASTV = N
+-                  DO LASTV = N, I+1, -1
+-                     IF( V( LASTV, I ).NE.ZERO ) EXIT
+-                  END DO
++C                 DO LASTV = N, I+1, -1
++C                    IF( V( LASTV, I ).NE.ZERO ) EXIT
++C                 END DO
```

Comment:
This is a pretty bad patch, that instead of actually rewriting this code in F77 style, just says "oh, we probably didn't need this anyway"